tests/onnxruntime/test_modeling.py

```diff
@@ -1445,6 +1449,175 @@ def test_compare_generation_to_io_binding(self, model_arch):
         gc.collect()
 
 
+class ORTModelForSpeechSeq2SeqIntegrationTest(unittest.TestCase):
+    SUPPORTED_ARCHITECTURES = ("whisper",)
+
+    def test_load_vanilla_transformers_which_is_not_supported(self):
+        with self.assertRaises(Exception) as context:
+            _ = ORTModelForSpeechSeq2Seq.from_pretrained(MODEL_NAMES["bert"], from_transformers=True)
+
+        self.assertIn("Unrecognized configuration class", str(context.exception))
+
+    @parameterized.expand(SUPPORTED_ARCHITECTURES)
+    def test_generate_utils(self, model_arch):
+        model_id = MODEL_NAMES[model_arch]
+        model = ORTModelForSpeechSeq2Seq.from_pretrained(model_id, from_transformers=True)
+        processor = get_preprocessor(model_id)
+
+        ds = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation")
+        data = ds[0]["audio"]["array"]
+        features = processor.feature_extractor(data, return_tensors="pt")
```

Comment:
Preprocessing for audio file requires installation of `soundfile` package. Should this be added to `setup.py` file? Or should we just generate dummy data for testing?

Reply:
Both are good IMO. You can add `soundfile` to the `testing` extra.